src/main/java/io/gravitee/common/utils/GraviteeConstants.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.common.utils;
+
+/**
+ * Gravitee ecosystem constants
+ */
+public class GraviteeConstants {
```

Comment:
Make it as `final`

Reply:
it has private constructor - so it is effectively final. But if you wish to - I'll do that. If you don't mind - after 2020-01-09. Is it ok?

Reply:
That's ok, I've missed the private constructor...
Please forget.